File: tests/fixtures/kotlin/input.kt
package geometry

interface Drawable {
    fun draw(): String
}

abstract class Shape(val area: Double) : Drawable {
    var name: String = ""

    abstract fun describe(): String

    override fun draw(): String {
        return "shape"
    }
}

fun greet(name: String): String {
    return "hi"
}

object Registry

File: tests/fixtures/php/input.php
<?php

namespace App\Geometry;

interface Drawable
{
    public function draw(): string;
}

class Shape implements Drawable
{
    private float $area;

    public function __construct(float $area)
    {
        $this->area = $area;
    }

    public function draw(): string
    {
        return "shape";
    }
}

function greet(string $name): string
{
    return "hi";
}
